apps/build.sh

```diff
@@ -58,6 +57,16 @@ if [ -f $pbc ] ; then
 fi
 
 docker_login
+buildx_create_args="--driver-opt image=moby/buildkit:${BUILDKIT_VERSION} --use"
+if [ -d /usr/local/share/ca-certificates ] ; then
+	# We need to pass these certs into the buildkit container
+	bkconf="/buildkit.toml"
```

Comment:
I suppose this file can be generating during the container image building (lmp-inage-tools).

Reply:
I don't think we want a custom image per instance.

Reply:
Is this directory `/usr/local/share/ca-certificates` of container mapped to a host directory?

Reply:
exactly - https://github.com/foundriesio/jobserv/commit/e7affa0eae6ae7ae8248bdb7cc7a8842f9430b44